### DISCUSSION TITLE
Rotate point in each side

### DISCUSSION THREAD

**Discussion by Elvin2019** [fabricjs/fabric.js#6876] [Q&A]
Hi dear team;

I would like to add rotation point (stick) to each side, is there any simple way to do that?
![fabric](https://user-images.githubusercontent.com/34287768/108412264-17a50c80-7243-11eb-92cc-c3f6faa993b5.PNG)

**Comment by asturur** [comment]
The stack overflow question was way better than this issue that does adhere to the bug type.

Please reopen the stack overflow question, and post a link to it in our discussion section.

**Comment by Elvin2019** [comment]
Thank you for your response, the link in below

[stack_overflow](https://stackoverflow.com/questions/66266893/fabric-js-rotating-point-in-each-side)

**Reply by asturur** [reply]
Your question has been rightfully flagged for low quality and closed.
I tried to improved it and asked for reopen, if they can keep it open i ll post the answer there.

**Comment by asturur** [comment]
You got an answer, but please try to learn to open better questions in general.
